lib/register_sources_dk/repositories/deltagerperson_repository.rb

```diff
@@ -66,6 +66,62 @@ def store(records)
         true
       end
 
+      def get_by_bods_identifiers(identifiers)
+        enheds_nummers = [] # enhedsNummer
+        cvrs = [] # virksomhedSummariskRelation.virksomhed.cvrNummer
+        identifiers.each do |identifier|
+          case identifier.schemeName
+          when 'Danish Central Business Register'
+            cvrs << identifier.id
+          when 'DK Centrale Virksomhedsregister'
+            enheds_nummers << identifier.id
+          end
+        end
+
+        return [] if cvrs.empty? && enheds_nummers.empty?
+
+        process_results(
+          client.search(
+            index:,
+            body: {
+              query: {
+                bool: {
+                  should: enheds_nummers.map { |id|
```

Comment:
It might be possible to use a single `bool`+`should`+`terms` query or similar rather than mapping to multiple conditions, but I haven't looked into it. I think it's okay for now, though, as it's consistent with the approach used elsewhere.